features/helpers/zypper.go

```diff
@@ -0,0 +1,84 @@
+package helpers
+
+import (
+	"encoding/xml"
+	"os/exec"
+	"testing"
+
+	"github.com/SUSE/connect-ng/internal/zypper"
+	"github.com/SUSE/connect-ng/pkg/registration"
+	"github.com/stretchr/testify/assert"
+)
+
+type Zypper struct {
+	t *testing.T
+}
+
+func NewZypper(t *testing.T) *Zypper {
+	return &Zypper{
+		t: t,
+	}
+}
+
+func (zypp *Zypper) run(params ...string) []byte {
+	cmd := exec.Command("zypper", params...)
+	result, err := cmd.CombinedOutput()
+
+	if err != nil {
```

Comment:
You may also want to cast this to `exec.ExitError` if it's non-nil. This way you can retrieve the exit code, which is relevant because zypper has special exit codes which might further help debugging errors (see the constants on `internal/zypper.zypper.go` named `zypperErr*` and `zypperInfo*`).